### PR TITLE
Wd smartseq multi sample scala

### PR DESCRIPTION
**Pull request by wjdingman1**

### CODE REVIEW

File: pipelines/skylab/smartseq2_multisample/MultiSampleSmartSeq2.changelog.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# 2.2.12
+2022-06-2022 (Date of Last Commit)
```
Comment (nikellepetrillo) [inline pipelines/skylab/smartseq2_multisample/MultiSampleSmartSeq2.changelog.md:2]:
```suggestion
2022-06-22 (Date of Last Commit)
```

File: pipelines/skylab/smartseq2_single_sample/SmartSeq2SingleSample.changelog.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+# 5.1.11
+2022-06-21 (Date of Last Commit)
+
+* Updated main workflow name from SmartSeq2SingleCell to SmartSeq2SingleSample in the SS2 single sample pipeline. This allows the pipeline to run in the updated scala tests.
```
Comment (nikellepetrillo) [inline pipelines/skylab/smartseq2_single_sample/SmartSeq2SingleSample.changelog.md:4]:
```suggestion
* Updated main workflow name from SmartSeq2SingleCell to SmartSeq2SingleSample in the SS2 single sample pipeline. This allows the pipeline to run in the updated scala tests.

```

**Review by nikellepetrillo — APPROVED** [review]
looks good!  DSP AppSec Trivy check (dockers/broad/rna_seq/fgbio) is failing but i dont think it has to do with your pr (transient?)

File: pipelines/skylab/smartseq2_single_sample/SmartSeq2SingleSample.changelog.md (hunk 2 above)
Comment (wjdingman1) [reply pipelines/skylab/smartseq2_single_sample/SmartSeq2SingleSample.changelog.md:4]: yeah the first PR needs to be merged then this will be fixed

**Comment by wjdingman1** [comment]
@nikellepetrillo yeah there is some weird network thing going on with trivy lately

**Pull request closed without merging**